docker-build.md

```diff
@@ -50,115 +47,151 @@ first half of this guide: [Exploring and Testing a Docker Container](docker-test
 ## 3. Create a Dockerfile
 
 A Dockerfile is a plain text file with keywords that add elements to a 
-Docker container. There are many keywords that can be used in a Dockerfile (documented on
+Docker image. There are many keywords that can be used in a Dockerfile (documented on
 the Docker website here: [Dockerfile
 keywords](https://docs.docker.com/engine/reference/builder/)), but we will use a 
 subset of these keywords following this basic outline: 
 
-- Starting point: Which Docker container do you want to start with?
+- Starting point: Which Docker image do you want to start with?
 - Additions: What needs to be added? Folders? Data? Other software? 
 - Environment: What variables (if any) are set as part of the software installation? 
 
 ### Create the file
 
 Create a blank text file. Because of 
-the way Docker builds containers, you should create a separate folder for each 
-new container you create with the appropriate Dockerfile inside. 
+the way Docker builds images, you should create a separate folder for each 
+new images you create with the appropriate Dockerfile inside. 
 You can call the file whatever you want, but the
```

Comment:
If you're removing the `-f` option from `docker build` below, this should be more emphatic about naming the file `Dockerfile`.

Reply:
Done!